### PR TITLE
Topic0.1

### PR DESCRIPTION
**Pull request by evannucci**

### CODE REVIEW

**Comment by evannucci** [comment]
Topic 0.1

File: 01-HTML-basics/exercise1_html/signup.html [hunk 1]
```diff
@@ -3,6 +3,9 @@
 	<head>
   		<meta name="description" content="Signup page">
  		<meta name="author" content="Ezequiel Vannucci">
+ 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+ 		<title>Signup</title>
 	</head>
```
Comment (pabloascencao) [inline 01-HTML-basics/exercise1_html/signup.html]: This form has 2 tags with the functionalyti `submit`, there's an input in line 25 and a button in line 39, there's should be just 1 of them.

**Review by pabloascencao — CHANGES_REQUESTED** [review]

File: 01-HTML-basics/exercise1_html/signup.html (hunk 1 above)
Comment (evannucci) [reply 01-HTML-basics/exercise1_html/signup.html]: Done !

File: 01-HTML-basics/exercise1_html/table.html [hunk 2]
```diff
@@ -3,6 +3,9 @@
 	<head>
   		<meta name="description" content="Table of expenses">
  		<meta name="author" content="Ezequiel Vannucci">
+ 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+ 		<title>Table</title>
 	</head>
```
Comment (pabloascencao) [inline 01-HTML-basics/exercise1_html/table.html]: If you want to add a `th` to the table is good to also add a `thead` and a `tbody` for the table body

**Review by pabloascencao — CHANGES_REQUESTED** [review]

File: 01-HTML-basics/exercise1_html/table.html (hunk 2 above)
Comment (evannucci) [reply 01-HTML-basics/exercise1_html/table.html]: Thead and tbody added

**Review by pabloascencao — APPROVED** [review]

**Pull request merged**